index.js

```diff
@@ -27,14 +27,13 @@ var formatOutput = function(report, file, options) {
     return err;
   });
 
-  var output = {
+  return {
```

Comment:
We can totally remove the previous formatting that I built in favor of CSSLint's, I think.